src/data/markdown/docs/02 javascript api/07 k6-experimental/06 webcrypto/02 SubtleCrypto.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'SubtleCrypto'
+excerpt: 'SubtleCrypto offers low-level cryptographic functions.'
+---
+
+The `SubtleCrypto` interface provides a set of low-level cryptographic primitives such as encryption, decryption, digital signature generation and verification, and key generation and management. It is useful for using secure and efficient cryptographic operations within k6 scripts, without requiring to have a deep understanding of the underlying cryptographic algorithms and protocols.
+
+## Methods
+
+| Method                                                                            | Description                                                                                 |
+| :-------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------ |
+| [encrypt](/javascript-api/k6-experimental/webcrypto/subtlecrypto/encrypt)         | Encrypts the given plaintext data using the specified algorithm and key.                    |
+| [decrypt](/javascript-api/k6-experimental/webcrypto/subtlecrypto/decrypt)         | Decrypts the given ciphertext data using the specified algorithm and key.                   |
+| [sign](/javascript-api/k6-experimental/webcrypto/subtlecrypto/sign)               | Signs the given data using the specified algorithm and key.                                 |
+| [verify](/javascript-api/k6-experimental/webcrypto/subtlecrypto/verify)           | Verifies the signature of the given data using the specified algorithm and key.             |
+| [digest](/javascript-api/k6-experimental/webcrypto/subtlecrypto/digest)           | Computes the digest (hash) of the given data using the specified algorithm.                 |
+| [generateKey](/javascript-api/k6-experimental/webcrypto/subtlecrypto/generatekey) | Generates a new cryptographic key for use with the specified algorithm.                     |
+| [importKey](/javascript-api/k6-experimental/webcrypto/subtlecrypto/importkey)     | Imports a raw key material into the Web Crypto API, generating a new key object that can be |used with the specified algorithm.
+| [exportKey](/javascript-api/k6-experimental/webcrypto/subtlecrypto/exportkey)     | Exports the raw key material of the given key object.                                       |
```

Comment:
Linking the MDN doc would be enough here, IMHO. I hope that we will have the capacity to keep it up-to-date.

Reply:
I actually considered doing that, but I rather disagree in the end. Having our own documentation pages leads to a better user-experience IMHO, which is something we should and do strive for first. Also in terms of SEO, I think it's better to not link-out, but that's really a side-thing.